shell/platform/embedder/embedder.h

```diff
@@ -963,6 +963,43 @@ typedef const FlutterLocale* (*FlutterComputePlatformResolvedLocaleCallback)(
     const FlutterLocale** /* supported_locales*/,
     size_t /* Number of locales*/);
 
+/// Display refers to a graphics hardware system consisting of a framebuffer,
+/// typically a monitor or a screen. This ID is unique per display and is
+/// stable until the Flutter application restarts.
+typedef uint64_t FlutterDisplayId;
+
+typedef struct {
+  /// This size of this struct. Must be sizeof(FlutterDisplay).
+  size_t struct_size;
+
+  FlutterDisplayId display_id;
+
+  /// A double-precision floating-point value representing the actual refresh
+  /// period in seconds. This value may be zero if the device is not running or
+  /// unavaliable or unknown.
+  double refresh_rate;
+
+} FlutterDisplay;
+
+typedef struct {
+  /// This size of this struct. Must be sizeof(FlutterDisplaySettings).
+  size_t struct_size;
+
+  /// Contains the actual number of displays returned in the displays array.
+  size_t display_count;
+
+  /// A pointer to storage provided by the caller for an array of
+  /// FlutterDisplays. On return, the array contains a list of active displays.
+  /// If you pass NULL, on return the display_count contains the total number of
```

Comment:
Again, this seems to be describing a function, but it's the comment for a struct element.

Reply:
Yup, agreed, I will go over the docs and make sure they align with structs and functions.